### DISCUSSION TITLE
封面是否可以加一页额外中文介绍页

### DISCUSSION THREAD

**Discussion by zhoujian9410** [nju-lug/NJUThesis#112] [Ideas]
现在这个封面会产生一张空白页，如下图：
![当前封面](https://raw.githubusercontent.com/zhoujian9410/zhoujian9410.github.io/master/%E5%BD%93%E5%89%8D%E5%B0%81%E9%9D%A2.png)

我的想法是，既然会有一张空白页，可以直接将其改为额外中文介绍页，如下图我师兄的论文
![师兄封面](https://raw.githubusercontent.com/zhoujian9410/zhoujian9410.github.io/master/%E5%B8%88%E5%85%84%E5%B0%81%E9%9D%A2.png)

不知道是否可以？以及工作量如何

ps：设置的是博士论文模板

**Comment by atxy-blip** [comment]
为了使用面向对象思想简化封面绘制，`njuthesis` 采用 `xtemplate` 宏包对页面进行了抽象化。目前研究生模板的 `\maketitle` 基本等价于替换为下面的命令：

```TeX
\UseInstance{nju}{cover-g-front-default}
\UseInstance{nju}{cover-g-back-default}
\UseInstance{nju}{cover-g-en-default}
```

其中每个 instance 对应一个页面对象，上面三条命令分别绘制了正面、背面、英文封面。

所以，如果要加一页，就是定义一个新的页面对象：

```TeX
\UseInstance{nju}{cover-g-front-default}
\UseInstance{nju}{cover-g-back-default}
\UseInstance{nju}{cover-g-info} # 自己定义的
\UseInstance{nju}{cover-g-en-default}
```

页面对象（page）的定义方法类似于
```TeX
\DeclareInstance { nju } { cover-g-info } { page }
  {
    element  = { title, author },
    prefix   = cover / g / info /,
    top-skip = 0 pt plus 1.2 fil
  }
```

其中每个页面元素对象（element）的定义方法类似于
```TeX
\DeclareInstance { nju } { cover / g / info / title } { page }
  {
    content     = 标题,
    format      = \zihao { 1 },
    bottom-skip = 1 cm
  }
```

介绍得比较粗糙，更详细的说明可以看手册中 5.4 节。如果你不喜欢在这种写法，也可以用其他方式在上述”自己定义的“位置加代码。

------

如果上面这个页面有普遍意义，可以保存成一个通用配置文件，利用不久前新增的 `config={...}` 这个文档类选项载入并重定义 `\maketitle`，不过那个估计还得我整（咕）

**Reply by zhoujian9410** [reply]
谢谢解答，我学习一下先
